tfhe/src/core_crypto/fft_impl/fft64/crypto/bootstrap.rs

```diff
@@ -243,54 +244,61 @@ impl<'a> FourierLweBootstrapKeyView<'a> {
         let lut_poly_size = lut.polynomial_size();
         let ciphertext_modulus = lut.ciphertext_modulus();
         assert!(ciphertext_modulus.is_compatible_with_native_modulus());
-        let monomial_degree = pbs_modulus_switch(
+        let monomial_degree = MonomialDegree(pbs_modulus_switch(
             *lwe_body,
             lut_poly_size,
             ModulusSwitchOffset(0),
             LutCountLog(0),
-        );
+        ));
 
         lut.as_mut_polynomial_list()
             .iter_mut()
             .for_each(|mut poly| {
-                polynomial_wrapping_monic_monomial_div_assign(
-                    &mut poly,
-                    MonomialDegree(monomial_degree),
-                )
+                let (mut tmp, _) = stack
+                    .rb_mut()
+                    .make_aligned_raw(poly.as_ref().len(), CACHELINE_ALIGN);
```

Comment:
can we add an update to the scratch function to capture this buffer being used ? I know it's going to be smaller than the size remaining at that point in the function but I would feel better if the scratch function reflected this new usage of the stack (if it's not too complicated to express) and if it's too complicated to express then I would like a comment in the scratch function or a check to verify it will work as intended if possible